lib/Dialect/Triton/Transforms/Combine.cpp

```diff
@@ -204,7 +204,7 @@ class CombineOpsPass : public TritonCombineOpsBase<CombineOpsPass> {
     patterns.add<CombineAddPtrPattern>(context);
     patterns.add<CombineBroadcastMulReducePattern>(context);
 
-    if (applyPatternsAndFoldGreedily(m, std::move(patterns)).failed())
+    if (applyPatternsGreedily(m, std::move(patterns)).failed())
```

Comment:
I don't think this is an equivalent translation to the new API. We should be specify fold=true through the rewrite options.